tests/get-cached-path.phpt

```diff
@@ -0,0 +1,39 @@
+--TEST--
+Test getCachedPath function
+
+--FILE--
+<?php
+
+require __DIR__ . "/../Patchwork.php";
+
+use Patchwork\CodeManipulation\Stream;
+
+$cachePath = str_replace('\\', '/', __DIR__ . '/cache');
+Patchwork\CodeManipulation\State::$cacheIndexFile = null;
+Patchwork\Config\State::$cachePath = $cachePath;
+
+if ( ! is_dir($cachePath)) {
+    mkdir($cachePath);
+}
```

Comment:
Personally, I would like to enforce an assumption that the tests leave the file system unaltered.

```suggestion
} else {
    echo "The cache directory should not exist in advance.\n";
}
```